src/Sequence Labelling/NER_DataDeps.jl

```diff
@@ -0,0 +1,29 @@
+using DataDeps
```

Comment:
Dont add a subfolder with a space. Also no capitalisation in the folder name. Change the folder name to `sequence` maybe?

Also, file names should be all small.

Reply:
Done.